src/Sulu/Bundle/ContactBundle/Contact/ContactManager.php

```diff
@@ -394,7 +395,7 @@ public function getById($id, $locale)
     {
         $contact = $this->contactRepository->find($id);
         if (!$contact) {
-            return;
+            throw new EntityNotFoundException($this->contactRepository->getClassName(), $id);
```

Comment:
Actually we don't throw this exception in the contact manager, since this is a REST exception, and the `ContactManager` might also be used in a different context...